iam_openapi_spec.yml

```diff
@@ -4677,13 +4678,17 @@ components:
       required:
         - domain
         - redirectUri
+        - email
```

Comment:
Can this be optional?

Reply:
we are sending login hint for all scenarios even though it is not supported in idP side.
FE is getting the email address in staging today. Good to keep this as a required parameter.